src/tests/test_verify_schema.sh

```diff
@@ -1,23 +1,44 @@
 #!/bin/bash -e
 
-PYTHONPATH="$PYTHONPATH:$TRAVIS_BUILD_DIR/src"
+# Use the Travis build dir if specified, otherwise assume that the
+# repo is unpacked in the current working dir
+REPO_ROOT_DIR=${TRAVIS_BUILD_DIR:-.}
+
+PYTHONPATH="$PYTHONPATH:$REPO_ROOT_DIR/src"
 export PYTHONPATH
 
 function verify_xml {
     # Verifies XML data against the schema
     xml="$1"
     type="$2"
     echo "Validating $type XML schema..."
-    xmllint --noout --schema "$TRAVIS_BUILD_DIR/src/schema/$type.xsd" $xml
+    xmllint --noout --schema "$REPO_ROOT_DIR/src/schema/$type.xsd" $xml
 }
 
-if [[ "$TRAVIS_PULL_REQUEST" == "false" ]]; then
+if [[ $TRAVIS_PULL_REQUEST == "false" ||
+      $GH_EVENT == 'push' &&
+      $GITHUB_REPOSITORY == 'opensciencegrid/topology' ]]; then
```

Comment:
When you mix `||` and `&&`, it's technically unambiguous (it's left-to-right since they have equal operator precedence), but it's a little less obvious which grouping you intended when i read it.

I'd suggest making it explicit, if this is the grouping you intended:

```suggestion
if [[ $TRAVIS_PULL_REQUEST == "false" || $GH_EVENT == 'push' ]] &&
   [[ $GITHUB_REPOSITORY == 'opensciencegrid/topology' ]]; then
```

(Or if you prefer, you can add parens inside the `[[ ... ]]`)

Reply:
(I'd also normally point out that `[[ $x == "string" ]]` is can equivalently be written `[[ $x = string ]]`, but that's pretty minor.)